jvm-packages/xgboost4j-spark/src/main/scala/ml/dmlc/xgboost4j/scala/spark/XGBoost.scala

```diff
@@ -378,11 +378,13 @@ private object Watches {
     }
 
     val (trainIter1, trainIter2) = trainPoints.duplicate
-    val trainMatrix = new DMatrix(trainIter1, cacheDirName.map(_ + "/train").orNull)
+    val trainMatrix = new DMatrix(trainIter1, cacheDirName.map(_ + "/train").orNull,
+      params.get("batchSize").map(_.toString.toInt).getOrElse(32<<10))
```

Comment:
If we set default value at ```BoosterParams```, here we can simplify as: ```params.get("batchSize").toInt```. We should manage default value at a fixed place to make the maintenance more easy.

Reply:
sounds good, fixed.